GNUmakefile

```diff
@@ -41,13 +41,12 @@ cov:
 	gocov test ${GOFILES} | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test:
+test: dev
 	@./scripts/verify_no_uuid.sh
-	@env \
-		GOTAGS="${GOTAGS}" \
-		GOFILES="${GOFILES}" \
-		TESTARGS="${TESTARGS}" \
-		sh -c "'$(CURDIR)/scripts/test.sh'"
```

Comment:
We should nuke `test.sh` as part of this, too. There was `go install` command in there that supposedly sped things up for Travis, I think when we build multiple test binaries.